Tweak TS sprite mechs visuals

reaperrr opened the pull request:
- renames 'run' sequence to 'walk'
- replaces the no-longer-necessary WithInfantryBody on the Titan
- adjusts Tick rates of walk anims and Wolverine attack anim
- fixes barrel, armament offsets of Titan and disables bullet image on `120mm`

reaperrr committed:
Tweak TS sprite walker animations

- replace WithInfantryBody with WithFacingSpriteBody + WithMoveAnimation on Titan
- rename all mech 'run' anims to 'walk'
- tweak walk anim rates to better match movement speed/original
- tweak Wolverine attack anim to play a little faster

reaperrr committed:
Fix TS Titan barrel, armament offsets and disable projectile image

reaperrr commented:
Rebased.

While I was at it, I added another commit that does some clean-up on `WithVoxelWalkerBody` and tweaks the MMk2 animation rate.

GraionDilach commented:
If you're doing things like this, you should consider making WVWB pausableconditional.

reaperrr commented:
> While I was at it, I added another commit that does some clean-up on WithVoxelWalkerBody and tweaks the MMk2 animation rate.

Removed that commit again (will return as separate PR later) and rebased, so we can hopefully get this merged and out of the PR queue soon.

GraionDilach reviewed (approved):
:+1:
